src/components/InvertedFlatList/index.tsx

```diff
@@ -5,9 +5,7 @@ import BaseInvertedFlatList from './BaseInvertedFlatList';
 
 // This is adapted from https://codesandbox.io/s/react-native-dsyse
 // It's a HACK alert since FlatList has inverted scrolling on web
-function InvertedFlatList<T>({onScroll: onScrollProp = () => {}, ...props}: FlatListProps<T>, ref: ForwardedRef<FlatList>) {
-    const {contentContainerStyle} = props;
-
+function InvertedFlatList<T>({onScroll: onScrollProp = () => {}, contentContainerStyle = {}, ...props}: FlatListProps<T>, ref: ForwardedRef<FlatList>) {
```

Comment:
@ishpaul777 You don't need a default value for style props, I think it was better before (RN styles can be falsy, and RN handles that gracefully)